frontend/public/components/conditions.tsx

```diff
@@ -11,15 +10,15 @@ export const Conditions: React.SFC<ConditionsProps> = ({obj}) => {
       {condition.status}
     </div>
     <div className="col-xs-3 col-sm-3">
-      {condition.reason}
+      {condition.reason || '-'}
     </div>
-    <div className="col-xs-3 col-sm-5">
-      {condition.message}
+    {/* remove initial newline which appears in route messages */}
+    <div className="col-xs-3 col-sm-5 co-pre-line">
+      {condition.message ? condition.message.replace(/^.*?\r?\n/, '') : '-'}
```

Comment:
This should use _.trim instead of the regex

```
{_.trim(condition.message) || '-'}
```

Reply:
thanks, fixed